api/report_viewer_server.thrift

```diff
@@ -279,6 +285,93 @@ service codeCheckerDBAccess {
                                              2: i64 new_run_id,
                                              3: DiffType diff_type,
                                              4: ReportFilterList reportFilters)
-                                             throws (1: shared.RequestFailed requestError)
+                                             throws (1: shared.RequestFailed requestError),
+
+
+
+
+
+    // store checker run related data to the database
+    // by default updates the results if name was already found
+    // using the force flag removes existing analysis results for a run
+    i64  addCheckerRun(
```

Comment:
The comments should be full sentences.

Does the force parameter make sense in the new model? Maybe it should be removed?